File: OLSource1/v1/VS_officedev/codesnippet/CSharp/how-to--programmatically-sort-data-in-worksheets_2.cs
            this.fruitList.Range.Sort(
                this.fruitList.ListColumns[1].Range, Excel.XlSortOrder.xlAscending,
                this.fruitList.ListColumns[2].Range, missing, Excel.XlSortOrder.xlAscending,
                missing, Excel.XlSortOrder.xlAscending, 
                Excel.XlYesNoGuess.xlYes, missing, missing, 
                Excel.XlSortOrientation.xlSortColumns,
                Excel.XlSortMethod.xlPinYin, 
                Excel.XlSortDataOption.xlSortNormal,
                Excel.XlSortDataOption.xlSortNormal,
                Excel.XlSortDataOption.xlSortNormal);

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/how-to--programmatically-sort-data-in-worksheets_2.vb
        Me.fruitList.Range.Sort( _
            Key1:=Me.fruitList.ListColumns(1).Range, Order1:=Excel.XlSortOrder.xlAscending, _
            Key2:=Me.fruitList.ListColumns(2).Range, Order2:=Excel.XlSortOrder.xlAscending, _
            Orientation:=Excel.XlSortOrientation.xlSortColumns, _
            Header:=Excel.XlYesNoGuess.xlYes)